app/components/CollapsibleDisplayContent/index.tsx

```diff
@@ -0,0 +1,92 @@
+import Editor from '@webkom/lego-editor';
+import '@webkom/lego-editor/dist/style.css';
+import 'react-image-crop/dist/ReactCrop.css';
+import { type CSSProperties, useRef, useState } from 'react';
+import Icon from '../Icon';
+import styles from './CollapsibleDisplayContent.css';
+
+type Props = {
+  /** The content to be displayed - the text */
+  content: string;
+
+  /** The id of the div wrapping the content - the id */
+  id?: string;
+
+  /** The classname of the div wrapping the content - the className */
+  className?: string;
+
+  /** Any style tp be added to the div wrapping the content - the style */
+  style?: CSSProperties;
+  placeholder?: string;
+  collapsedHeight?: number;
+};
+
+/**
+ * Renders `content` produced by the Editor component in a read-only format.
+ */
+function CollapsibleDisplayContent({
```

Comment:
I'd prefer a short name, like `CollapsibleContent`. Not sure why "display" is needed (I get that it was named that from before).

Reply:
I understand the use of `Display` as the content being "read-only", though there might exist better names for it ...

Reply:
I'm not following you on the "read-only" part. `CollapsibleContent` is much cleaner and more precise.

Reply:
It makes more sense if you look at how it is implemented: It uses the `Editor`-component to display the content, but since you cannot edit it, it is only a "display".

I'd prefer keeping the name as it is such that it is easy to see it is related to the `DisplayContent` and not the `Collapse`-wrapper (which we also have but isn't flexible enough for this purpose unfortunately).